src/client/app/components/admin/PreferencesComponent.tsx

```diff
@@ -358,6 +391,16 @@ export default function PreferencesComponent() {
 					onChange={e => makeLocalChanges('defaultHelpUrl', e.target.value)}
 				/>
 			</div>
+			<div>
+				<p className='mt-2' style={titleStyle}>
+					<FormattedMessage id='default.weather.location' />:
+				</p>
+				<Input
+					type='text'
```

Comment:
I think this should be a drop down menu with a list of all the known weather locations for this site (or empty if none). This is similar to the timezone above or on a number of other pages.